{
  "uid" : "2eeb634a519bd79f",
  "name" : "User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "fullName" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT: User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "historyId" : "3d26ee416cc3affe0103c3d8164ab3c5",
  "time" : {
    "start" : 1757921091226,
    "stop" : 1757921097980,
    "duration" : 6754
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
  "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
    "steps" : [ {
      "name" : "Given systems up",
      "time" : {
        "start" : 1757921091227,
        "stop" : 1757921092462,
        "duration" : 1235
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "Given a cart of RPTs non-multibeneficiary",
      "time" : {
        "start" : 1757921092463,
        "stop" : 1757921092463,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1757921092463,
        "stop" : 1757921092463,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1757921092463,
        "stop" : 1757921092463,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false,
      "stepsCount" : 0
    }, {
      "name" : "And an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT",
      "time" : {
        "start" : 1757921092463,
        "stop" : 1757921093148,
        "duration" : 685
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "acaa21a6ab4d243f",
        "name" : "Step: [an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT] - Sent request",
        "source" : "acaa21a6ab4d243f.txt",
        "type" : "text/plain",
        "size" : 1260
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true,
      "stepsCount" : 0
    }, {
      "name" : "When the user tries to pay a cart of RPTs on EC website",
      "time" : {
        "start" : 1757921093148,
        "stop" : 1757921094042,
        "duration" : 894
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "dd3fad3665626517",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Sent request",
        "source" : "dd3fad3665626517.txt",
        "type" : "text/plain",
        "size" : 15123
      }, {
        "uid" : "7923bc275b3f2aae",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Received response",
        "source" : "7923bc275b3f2aae.txt",
        "type" : "text/plain",
        "size" : 409
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true,
      "stepsCount" : 0
    }, {
      "name" : "Then conversion to new model fails in wisp-converter",
      "time" : {
        "start" : 1757921094042,
        "stop" : 1757921095050,
        "duration" : 1008
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "e32938bc48e01fe8",
        "name" : "Step: [conversion to new model fails in wisp-converter]: on sessionId: [e23581c2-1491-42ed-b2d3-433e52ed75ac] - Received response",
        "source" : "e32938bc48e01fe8.txt",
        "type" : "text/plain",
        "size" : 5807
      }, {
        "uid" : "49be5f895f2720d1",
        "name" : "Received response",
        "source" : "49be5f895f2720d1.attach",
        "type" : "text/html",
        "size" : 5677
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true,
      "stepsCount" : 0
    }, {
      "name" : "And the KO receipt is sent",
      "time" : {
        "start" : 1757921095050,
        "stop" : 1757921097980,
        "duration" : 2930
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "36e6d7ef92041241",
        "name" : "Step: [the KO receipt is sent]: on IUV: [192330062280390] - Received response",
        "source" : "36e6d7ef92041241.txt",
        "type" : "text/plain",
        "size" : 55789
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true,
      "stepsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : false,
    "hasContent" : true,
    "stepsCount" : 8
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "nodo_invia_carrello_rpt"
  }, {
    "name" : "tag",
    "value" : "runnable"
  }, {
    "name" : "tag",
    "value" : "unhappy_path"
  }, {
    "name" : "feature",
    "value" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "runnable", "nodo_invia_carrello_rpt", "unhappy_path" ]
  },
  "source" : "2eeb634a519bd79f.json",
  "parameterValues" : [ ]
}